extras/parse_travis_log.py

```diff
@@ -241,26 +269,40 @@ def print_static_analysis(d):
             if line.strip() == 'WARNING: Not uploading results as this is a pull request':
                 uploading_results = False
 
+            if 'Skipping a deployment with the s3 provider because' in line:
+                uploading_results = False
+
+    # post process values
+    actual_warnings = { k.split("nest-simulator/")[1] :v for k,v in warnings.iteritems() if k.startswith('/home/travis/build') }
```

Comment:
I don't get this. From the initialization of "warnings" and here, it seems like it is supposed to be a dict, but in the code above it is assigned a return value from count_warnings_errors, which is an int. Have I missed something?